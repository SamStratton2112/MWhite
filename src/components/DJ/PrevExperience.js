import { Container, Row, Col } from 'react-bootstrap';
import mike from '../../data.js';
import LocationCard from './LocationsCard.js';

const PrevExperience =()=>{
    return(
        <Container className='center containerBckground'>
            <Row md={12}>
                <Col>
                    <LocationCard location={mike.dj.prevExperience.sanDiego}/>
                </Col>
                <Col>
            <LocationCard location={mike.dj.prevExperience.Vegas}/>
                </Col>
                <Col>
            <LocationCard location={mike.dj.prevExperience.la}/>
                </Col>
                <Col>
            <LocationCard location={mike.dj.prevExperience.palmSprings}/>
                </Col>
                <Col>
            <LocationCard location={mike.dj.prevExperience.ny}/>
                </Col>
                <Col>
            <LocationCard location={mike.dj.prevExperience.Charlotte}/>
                </Col>
            </Row>
        </Container>
    )
}

export default PrevExperience;