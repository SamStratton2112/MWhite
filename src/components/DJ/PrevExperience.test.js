import PrevExperience from './PrevExperience';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';

describe(PrevExperience, () => {
    it("renders without crashing", () => {
        render(<MemoryRouter>
            <PrevExperience />
        </MemoryRouter>);
    });
    it("matches snapshot", () => {
        const { asFragment } = render(<MemoryRouter>
            <PrevExperience />
        </MemoryRouter>);
        expect(asFragment()).toMatchSnapshot();
    });
});