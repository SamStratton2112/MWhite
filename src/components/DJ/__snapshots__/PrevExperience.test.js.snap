// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrevExperience matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="center container"
  >
    <div
      class="row row-cols-lg-3 row-cols-md-12"
    >
      <div
        class="col"
      >
        <div
          class="col"
        >
          <p
            style="border-radius: .2em; margin-top: 1em; width: 12em;"
          >
            San Diego, California
          </p>
          <div
            class="container"
            style="width: 20rem; height: 20rem;"
          >
            <img
              alt="San Diego, California"
              src="https://i.imgur.com/2O7KGP6.jpeg"
              style="width: 20rem; height: 20rem; border-radius: .5em;"
            />
          </div>
        </div>
      </div>
      <div
        class="col"
      >
        <div
          class="col"
        >
          <p
            style="border-radius: .2em; margin-top: 1em; width: 12em;"
          >
            Las Vegas, Nevada
          </p>
          <div
            class="container"
            style="width: 20rem; height: 20rem;"
          >
            <img
              alt="Las Vegas, Nevada"
              src="https://i.imgur.com/WYPhHOV.jpeg"
              style="width: 20rem; height: 20rem; border-radius: .5em;"
            />
          </div>
        </div>
      </div>
      <div
        class="col"
      >
        <div
          class="col"
        >
          <p
            style="border-radius: .2em; margin-top: 1em; width: 12em;"
          >
            Los Angeles, California
          </p>
          <div
            class="container"
            style="width: 20rem; height: 20rem;"
          >
            <img
              alt="Los Angeles, California"
              src="https://i.imgur.com/uLfW5z0.png"
              style="width: 20rem; height: 20rem; border-radius: .5em;"
            />
          </div>
        </div>
      </div>
      <div
        class="col"
      >
        <div
          class="col"
        >
          <p
            style="border-radius: .2em; margin-top: 1em; width: 12em;"
          >
            Palm Springs, California
          </p>
          <div
            class="container"
            style="width: 20rem; height: 20rem;"
          >
            <img
              alt="Palm Springs, California"
              src="https://i.imgur.com/soZ7oyi.jpeg"
              style="width: 20rem; height: 20rem; border-radius: .5em;"
            />
          </div>
        </div>
      </div>
      <div
        class="col"
      >
        <div
          class="col"
        >
          <p
            style="border-radius: .2em; margin-top: 1em; width: 12em;"
          >
            New York, New York
          </p>
          <div
            class="container"
            style="width: 20rem; height: 20rem;"
          >
            <img
              alt="New York, New York"
              src="https://i.imgur.com/3JBkh6t.jpeg"
              style="width: 20rem; height: 20rem; border-radius: .5em;"
            />
          </div>
        </div>
      </div>
      <div
        class="col"
      >
        <div
          class="col"
        >
          <p
            style="border-radius: .2em; margin-top: 1em; width: 12em;"
          >
            Charlotte, North Carolina
          </p>
          <div
            class="container"
            style="width: 20rem; height: 20rem;"
          >
            <img
              alt="Charlotte, North Carolina"
              src="https://i.imgur.com/Oqdr1Ty.jpeg"
              style="width: 20rem; height: 20rem; border-radius: .5em;"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
